src/test/emulators/auth/signUp.spec.ts

```diff
@@ -470,20 +489,29 @@ describeAuthEmulator("accounts:signUp", ({ authApi }) => {
       });
   });
 
-  it("should error if a multi factor enrollment ID is provided on create", async () => {
-    const mfaInfo = {
-      displayName: "Cell Phone",
-      phoneInfo: TEST_PHONE_NUMBER,
-      mfaEnrollmentId: randomId(28),
+  it("should ignore if multi factor enrollment ID is specified on create", async () => {
+    const mfaEnrollmentId1 = randomId(28);
```

Comment:
Let's hard code some IDs like `thisShouldBeIgnored1` -- more readable, and the test won't randomly fail if we had really really bad (good?) luck with the RNG.